sim/midas/src/main/cc/emul/verilator-harness.cc

```diff
@@ -183,6 +209,38 @@ void tick() {
                           top->dma_r_valid,
                           top->dma_b_bits_id,
                           top->dma_b_valid);
+#endif // DMA_PRESENT
+
+#ifdef FPGA_MANAGED_AXI4_PRESENT
+  simif_emul_t::cpu_mem->tick(top->reset,
+                              top->fmaxi4_ar_valid,
+                              top->fmaxi4_ar_bits_addr,
+                              top->fmaxi4_ar_bits_id,
+                              top->fmaxi4_ar_bits_size,
+                              top->fmaxi4_ar_bits_len,
+
+                              top->fmaxi4_aw_valid,
+                              top->fmaxi4_aw_bits_addr,
+                              top->fmaxi4_aw_bits_id,
+                              top->fmaxi4_aw_bits_size,
+                              top->fmaxi4_aw_bits_len,
+
+                              top->fmaxi4_w_valid,
+#if FPGA_MANAGED_AXI4_STRB_BITS > 64
+                              &top->fmaxi4_w_bits_strb,
```

Comment:
Can you remind me why `top` silently converts from `&top` to `top`?

Reply:
The type that verilator uses to represent a bitvector changes for bitwidths > 64. This would be more clear as &(top-> fmaxi4_w_bits_strb).